.changeset/strange-flies-retire.md

```diff
@@ -0,0 +1,6 @@
+---
+'@keystonejs/keystone': minor
+'@keystonejs/test-utils': patch
+---
+
+Moved adapter mocks into test utils package. No functional changes but if running tests update keystone and test-utils packages.
```

Comment:
```suggestion
Moved adapter mocks into test utils package.
```

This comment isn't relevant to users of keystone, as we don't include the tests in our published packages.